cli/command/formatter/container.go

```diff
@@ -115,7 +116,22 @@ func (c *containerContext) Image() string {
 		if trunc := stringid.TruncateID(c.c.ImageID); trunc == stringid.TruncateID(c.c.Image) {
 			return trunc
 		}
+		// truncate digest if no-trunc option was not selected
+		ref, err := reference.ParseNormalizedNamed(c.c.Image)
+		if err == nil {
+			if nt, ok := ref.(reference.NamedTagged); ok {
+				// case for when a tag is provided
+				if namedTagged, err := reference.WithTag(reference.TrimNamed(nt), nt.Tag()); err == nil {
+					return reference.FamiliarString(namedTagged)
+				}
+			} else {
+				// case for when a tag is not provided
+				named := reference.TrimNamed(ref)
+				return reference.FamiliarString(named)
```

Comment:
Thinking about this some more, I'm not sure we want to discard the digest in this case.

It makes sense for the digest+tag case, because the digest is primarily internal information that doesn't need to clutter the output.

But if there's only a digest, it feels like it might be misleading to only show the name.

Not sure what others think. If there's a good UX reason to never show digests here, I guess it's okay.